src/components/Atoms/Controls/Input/index.tsx

```
@@ -167,112 +178,146 @@ const Input = React.forwardRef<HTMLInputElement, CustomInputProps>(
+        switch: (
+          <Switch
+            disabled={disabled}
+            name={name}
+            label={label}
+            className={className}
+            {...props}
+            ref={(e) => {
+              if (e) {
+                inputRef(e);
+                if (typeof ref === "function") ref(e);
+                else if (ref) (ref as React.MutableRefObject<any>).current = e;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)